themes/default/layouts/shortcodes/cli-note.html

```diff
@@ -5,11 +5,10 @@
     </div>
     <div class="note-body">
         <p>
-            If this is your first time running <code>pulumi new</code> or most other <code>pulumi</code> commands, you will be prompted to log in to the
-            <a href="https://app.pulumi.com" target="_blank">Pulumi service</a>. The <a href="{{ relref . "/docs/reference/cli" }}">Pulumi CLI</a> works in tandem with the Pulumi
-            service in order to deliver a reliable experience. It is free for individual use, with
-            <a href="{{ relref . "/pricing" }}" target="_blank"> features available for teams</a>. Hitting <code>ENTER</code> at the prompt opens up a web browser allowing you to
-            either sign in or sign up.
+            If this is your first time running <code>pulumi new</code> or other <code>pulumi</code> commands, you may be prompted to sign in to the
+            <a href="https://app.pulumi.com" target="_blank">Pulumi Service</a>. The Pulumi <a href="{{ relref . "/docs/reference/cli" }}">CLI</a> and Service
+            work in tandem to deliver a reliable experience. It's free for individual use, with <a href="{{ relref . "/pricing" }}">features available for teams</a>, and
+             <a href="{{ relref . "/docs/intro/concepts/state" }}">self-managed options</a> are also available. Hitting Enter at the prompt opens a browser for you to sign in or sign up.
```

Comment:
- pedantically speaking the cli uses `log in` not `sign in`.
- could we keep `Pulumi` in the CLI link? and also include it in the Service mention?

Reply:
> pedantically speaking the cli uses log in not sign in

Yeah, despite that you "sign in" to the Service pretty much everywhere, sticking with "login" or "log in" at the CLI level since that maps to the command seems like a fine thing to me. Will tweak.

> could we keep Pulumi in the CLI link? and also include it in the Service mention

Ah yes, that was unintentional -- I'll fix.